### PR TITLE
fix: misplaced modal in <aside> [WPB-6844]

### PR DESCRIPTION
**Pull request by aweiss-dev**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-6844" title="WPB-6844" target="_blank">BugWPB-6844</a>  E2EI certificate details modal should be shown fullscreen
  </td></table>
  <br />
 

The custom modal placement inside the "aside - nav" was suboptimal and led to the bug mentioned in the ticket.

I changed the implementation to use our PrimaryModal to display the information instead.

The "design change" is aligned with Wolfgang.

BEFORE:

Screenshot 2024-02-26 at 10 28 09

AFTER:

Screenshot 2024-03-26 at 11 52 32
Screenshot 2024-03-26 at 11 52 36
Screenshot 2024-03-26 at 11 52 41

### CODE REVIEW

File: src/script/components/Modals/PrimaryModal/Components/MessageContent.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
  * Wire
```
Comment (aweiss-dev) [inline src/script/components/Modals/PrimaryModal/Components/MessageContent.tsx]: This makes no sense git!

One file was deleted and another newly created, and git mixed them up :D

File: src/script/page/MainContent/panels/preferences/DevicesPreferences/components/E2EICertificateDetails/useCertificateDetailsModal.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      letterSpacing: '0.05px',
+      lineHeight: 'var(--line-height-md)',
+      wordBreak: 'break-word',
+      // The margin is set to -30px to compensate for the margin of the modal content
+      marginBottom: '-30px',
+    }}
+  >
+    <p>{message}</p>
+  </div>
+);
```
Comment (V-Gira) [inline src/script/page/MainContent/panels/preferences/DevicesPreferences/components/E2EICertificateDetails/useCertificateDetailsModal.tsx]: Why are we getting rid of the `.style.ts` file?

Comment (aweiss-dev) [reply src/script/page/MainContent/panels/preferences/DevicesPreferences/components/E2EICertificateDetails/useCertificateDetailsModal.tsx]: Was not needed in this case. I don't see a benefit in placing styles for this component in another file. This particular style is tailored to this particular component and, therefore, should be placed right next to (or into) it.

File: src/script/page/MainContent/panels/preferences/DevicesPreferences/components/E2EICertificateDetails/useCertificateDetailsModal.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+import {downloadFile} from 'Util/util';
+
+const COPY_MESSAGE_TIMEOUT = 3000;
+const DOWNLOAD_CERTIFICATE_TIMEOUT = 500;
+const CERTIFICATE_NAME = 'certificate.pem';
+const CERTIFICATE_TYPE = 'application/x-pem-file';
+
+const ModalMessageWrapper = ({message}: {message: string}) => (
+  <div
+    style={{
```
Comment (atomrc) [inline src/script/page/MainContent/panels/preferences/DevicesPreferences/components/E2EICertificateDetails/useCertificateDetailsModal.tsx]: This should be `css` not inline styles here

**Review by atomrc — APPROVED** [review]

**Review by atomrc — APPROVED** [review]

**Pull request merged**